plugins/outputs/stackdriver/sample.conf

```
@@ -3,6 +3,15 @@
   ## GCP Project
   project = "erudite-bloom-151019"
 
+  ## Quota Project
+  ## Specifies the Google Cloud project that should be charged for metric ingestion quota.
+  ## If omitted, the quota is charged to the project associated with the service account.
+  ## This is useful when sending metrics to multiple projects using a single service account, 
+  ## ensuring that the target projects are billed instead of the service account's project.
+  ##
+  ## The caller must have the `serviceusage.services.use` permission on the specified quota project.
+  # quota_project = "billing-project"
+
```

I think the default is empty isn't it? So how about shortening this to
```suggestion
  ## Quota Project to charge for metric ingestion quota
  ## If omitted, the quota is charged to the project associated with the service account.
  ## The caller must have the `serviceusage.services.use` permission on the specified quota project.
  # quota_project = ""

```

---

Hey there! So I guess the only concern I would have is that we removed the explanation as to what this would be used for, sending to multiple projects using a single SA while pointing the quota at the destination project instead of the SA project. 

How about this?

```
## Quota Project
## Specifies the Google Cloud project that should be billed for metric ingestion.
## If omitted, the quota is charged to the service account’s default project.
## This is useful when sending metrics to multiple projects using a single service account.
## The caller must have the `serviceusage.services.use` permission on the specified project.
# quota_project = ""
```

---

Yeah I like this!